packages/selector-ui/src/components/wallet-selector/readme.md

```diff
@@ -0,0 +1,46 @@
+# wallet-selector
+
+
+
+<!-- Auto Generated Below -->
+
+
+## Methods
+
+### `setRouteName(route: string) => Promise<void>`
```

Comment:
Not sure if we need this method?

Reply:
I was thinking it might be useful and it was an easy way to reset routeName to "WalletOptions" when closing the modal.
I will remove this and probably save routeName in store(state).

Reply:
As it is now it has the same logic as before but I will try to remove that the setRouteName method.
We used to have a useEffect to check the state for the `visible `variable, not sure how to do it in stencil but I will try to implement a similar solution.